unittests/ahbs/FV2204/IFTSTA/21035_deep.json

```diff
@@ -19,7 +19,13 @@
                     {
```

Comment:
schaut bei diesen dateien auch gerne parallel ins AHB, ob alle einträge die bei 21035 (und entsprechend in den anderen files 11042 und 35001) ein X haben, hier auch noch einträge haben. es sollten nur die dictionary key/value pairs ersatzlos verloren gehen, die kein "X" haben.

Reply:
Bei der 21035 und der 35001 sollte gar nichts verloren gehen oder? Weil sie die einzige Spalte sind?

Reply:
hrm... bei #40 abe ich genau diese arugmentation verwendet. stimmt.